test/tests/setup.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+yum install nodejs npm
+npm -g install phantomjs
+
+yum install yum-plugin-copr
+yum copr enable lmr/Autotest
+yum install avocado
+yum install avocado-virt
+
+# because of troubles with cockpit recompiled to /opt/ instead of /usr
+setenforce 0
+
+mkdir -p /usr/share/avocado/tests/
+
+/bin/cp -rf  * /usr/share/avocado/tests/; python checklogin.py ; cat /root/avocado/job-results/latest/job.log 
```

Comment:
There is more possibilities:
1) in case of running tests on virt-machines,  setup part coul be part of installation process of vms, then there will be prepared snapshot with these dependencies and ther will be need only to install cockpit to version you want
2) In case of testing in openstack (it is same as 1.)
3) in case of running anywhere, thats why there is need cleanup part in every test, what cause, that this setup enviroment part will be scheduled only once time (at the be begin) and then for example 10 tests will be scheduled.